### ISSUE TITLE
Entire editor collapses to one line after enter, backspace.

### ISSUE THREAD

**Issue by Luke-SF** [tinymce/tinymce#2813]
As seen in the example here: http://fiddle.tinymce.com/Rtfaab
whenever you press enter a few times in an empty editor, then want to backspace a line, the entire editor blanks when you hit backspace.

```
<script type="text/javascript">
tinymce.init({
    selector: "div",
    plugins: [
        "advlist autolink lists link image charmap print preview anchor",
        "searchreplace visualblocks code fullscreen",
        "insertdatetime media table contextmenu paste"
    ],
    inline: true,
    toolbar: "insertfile undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image"
});
</script>

Delete the content in the editor, press enter a few times, then backspace and the entire editor is collapsed to 1 line.

<form method="post" action="dump.php">
    <div name="content">Deleteme</div>
</form>
```

**Comment by spocke** [comment]
Couldn't reproduce it in FF or Chrome. What browser is this happening in?

**Comment by Luke-SF** [comment]
Both Firefox and Chrome, probably others. Select and delete the "deleteme" text on that fiddle, then press enter a few times, then backspace.

**Comment by alecpl** [comment]
Confirmed issue using FF 45.

**Comment by Afraithe** [comment]
I can confirm this is still going on in Firefox, all <p>&nbsp;</p> are deleted on a single backspace hit.

**Comment by Luke-SF** [comment]
Still happening with TinyMCE 4.4.1 in Chrome 55 beta.

**Comment by mehronkugler** [comment]
I ran into this issue using 5.0.13 with Chrome 78.

I found the solution was to apply a `min-height` css property to the `mce-content-body` class. For example I gave my editor a min-height of 24 and that seemed to look nice. 

Even though the <br>, that sometimes appears inside the editor's paragraph content, is deleted somehow, adding a min-height attribute creates a space in which it is possible to click again and restore editing functions.
